test/python/transpiler/test_elide_permutations.py

```diff
@@ -312,7 +313,7 @@ def test_not_run_after_layout(self):
         qc.h(1)
 
         spm = generate_preset_pass_manager(
-            optimization_level=3, initial_layout=list(range(2, -1, -1)), seed_transpiler=42
+            optimization_level=1, initial_layout=list(range(2, -1, -1)), seed_transpiler=42
         )
         spm.layout += ElidePermutations()
```

Comment:
This was the other new failing test. Now that the `ElidePermutations` already runs in level=3, it is able to remove those SWAP gates before getting to the extra pass in line 318.